### PR TITLE
Remove gyro_scaled

### PR DESCRIPTION
**Pull request by haslinghuis**
- keep in sync with https://github.com/betaflight/betaflight/pull/13101

### CODE REVIEW

File: src/flightlog_fielddefs.js [hunk 1]
```diff
@@ -526,6 +526,10 @@ export function adjustFieldDefsList(firmwareType, firmwareVersion) {
     if (semver.lt(firmwareVersion, "4.6.0")) {
       DEBUG_MODE.splice(DEBUG_MODE.indexOf("D_MAX"), 1, "D_MIN");
     }
+    if (semver.gte(firmwareVersion, "4.7.0")) {
```
Comment (nerdCopter) [inline src/flightlog_fielddefs.js:529]: 4.7?

Comment (nerdCopter) [reply src/flightlog_fielddefs.js:529]: i see, i suppose this keep compatibility for earlier version of 4.6.

Comment (haslinghuis) [reply src/flightlog_fielddefs.js:529]: No you are right - used to 1.47 being 4.6 :)

File: src/flightlog_fields_presenter.js [hunk 2]
```diff
@@ -1308,16 +1308,27 @@ FlightLogFieldPresenter.adjustDebugDefsList = function (
         'debug[7]':'Notch 7 Center Freq [dbg-axis]',
       };
     }
+    if (semver.gte(firmwareVersion, '4.7.0')) {
```
Comment (nerdCopter) [inline src/flightlog_fields_presenter.js:1311]: here too?

**Comment by nerdCopter** [comment]
- this is proper, but also if a early 4.6 had gyro_scaled, then this PR does not display it.
- so what do we want to do? allow 4.6 with gyro_scaled, or cut the cord and move on?

  - here is a 4.6 BBL with unfiltered turn off and debug=gyro_scaled  for testing.
[BTFL_BLACKBOX_LOG_APEX-6INCH_20240421_151202_APEXF7_flown_fine_little_quivers_highWinds.BBL.zip](https://github.com/user-attachments/files/17524612/BTFL_BLACKBOX_LOG_APEX-6INCH_20240421_151202_APEXF7_flown_fine_little_quivers_highWinds.BBL.zip)
  - see empty
 
image

File: src/flightlog_fielddefs.js [hunk 3]
```diff
@@ -526,6 +526,10 @@ export function adjustFieldDefsList(firmwareType, firmwareVersion) {
     if (semver.lt(firmwareVersion, "4.6.0")) {
       DEBUG_MODE.splice(DEBUG_MODE.indexOf("D_MAX"), 1, "D_MIN");
     }
+    if (semver.gte(firmwareVersion, "4.6.0")) {
+      DEBUG_MODE.splice(DEBUG_MODE.indexOf('GPS_RESCUE_THROTTLE_PID'), 1, 'AUTOPILOT_ALTITUDE');
+      DEBUG_MODE.splice(DEBUG_MODE.indexOf("GYRO_SCALED"), 1);
```
Comment (nerdCopter) [inline src/flightlog_fielddefs.js:531]: maybe skip the gyro_scaled for now, merge only the autopilot_altitude :man_shrugging:  ??
```suggestion
```

do gyro_scaled in another PR and wait?

Comment (haslinghuis) [reply src/flightlog_fielddefs.js:530]: Firmware PR is ready to merge, just waiting for client support to be added first :upside_down_face:

**Review by nerdCopter — APPROVED** [review]
- approving for when ready.
- no rush

**Comment by haslinghuis** [comment]
Rebased and resolved merge conflict

**Review by nerdCopter — APPROVED** [review]
